src/main/java/uk/gov/pay/adminusers/service/InviteService.java

```diff
@@ -48,15 +48,18 @@ public InviteService(RoleDao roleDao,
                          InviteDao inviteDao,
                          AdminUsersConfig config,
                          PasswordHasher passwordHasher,
-                         NotificationService notificationService, SecondFactorAuthenticator secondFactorAuthenticator) {
+                         NotificationService notificationService,
```

Comment:
This constructor is growing and have the same symptoms to that of UserService. I would have preferred an approach similar to ServiceFactory approach (see UserServicesFactory.java) so that the business logic can scale horizontally.
Although I'm not too picky on this comment and happy to do this kind of refactoring at a later stage if its too much